fastlane/Fastfile

```diff
@@ -0,0 +1,61 @@
+# By Jakub Kaspar 21/11/2020
+# Copyright © 2020 Sequel-Ace. All rights reserved.
+
+# Fastlane Constants
+default_platform :mac
+fastlane_version "2.168.0"
+
+GITHUB_REPO = "Sequel-Ace/Sequel-Ace"
+PROJECT_NAME = "sequel-ace.xcodeproj"
+WORKSPACE_NAME = "sequel-ace.xcworkspace"
+TARGET_NAME = "Sequel Ace"
```

Comment:
Should this be the "Sequel Ace Release..." target? We have different build settings for each target type and I don't see this PR changing that anywhere?

Reply:
You are talking about Scheme, not Target - Target is only one. :)